pkg/aws/actuator/actuator.go

```diff
@@ -896,9 +896,14 @@ func (a *AWSActuator) getDesiredUserPolicy(entries []minterv1.StatementEntry, us
 	return string(b), nil
 }
 
+// GetParentCredSecretLocation returns the namespace and name where the parent credentials secret is stored.
+func (a *AWSActuator) GetParentCredSecretLocation() types.NamespacedName {
+	return types.NamespacedName{Namespace: constants.KubeSystemNS, Name: annotatorconst.AWSCloudCredSecretName}
+}
+
 func (a *AWSActuator) getCloudCredentialsSecret(ctx context.Context, logger log.FieldLogger) (*corev1.Secret, error) {
 	cloudCredSecret := &corev1.Secret{}
-	if err := a.Client.Get(ctx, types.NamespacedName{Name: rootAWSCredsSecret, Namespace: rootAWSCredsSecretNamespace}, cloudCredSecret); err != nil {
+	if err := a.Client.Get(ctx, types.NamespacedName{Name: annotatorconst.AWSCloudCredSecretName, Namespace: constants.KubeSystemNS}, cloudCredSecret); err != nil {
```

Comment:
instead of 'types.NamespacedName{Name: annotatorconst.AWSCloudCredSecretName, Namespace: constants.KubeSystemNS}' can we use a.GetParentCredSecretLocation()?

Reply:
Good catch, I didn't think of going back to this after adding the actuator method.